server/conn.go

```diff
@@ -1254,55 +1258,83 @@ func (cc *clientConn) handleIndexAdvise(ctx context.Context, indexAdviseInfo *ex
 // There is a special query `load data` that does not return result, which is handled differently.
 // Query `load stats` does not return result either.
 func (cc *clientConn) handleQuery(ctx context.Context, sql string) (err error) {
-	rss, err := cc.ctx.Execute(ctx, sql)
+	stmts, err := cc.ctx.Parse(ctx, sql)
```

Comment:
should we keep old `s.PrepareTxnCtx` call before here keep old logic parallel running parse with get tso

Reply:
Get TS future has been moved by  @crazycs520 
We don't run Parse and GetTS parallel now.
https://github.com/pingcap/tidb/pull/11981/files#diff-8d751b31bc68f275a366be85918effd0R45